mantidproject/mantid
# Added DET/OVERLAP option to user file parser

It was requested that the user file parsers be updated to allow the merge range to be set from the user file using the syntax. DET/OVERLAP Q1 Q2. 

**To test:**
- Download this data
[loqdemo.zip](https://github.com/mantidproject/mantid/files/1748425/loqdemo.zip)

- In Mantid open the old sans GUI ensure that the instrument is set to LOQ and load the maskfile from the folder. 
- Load the batch file from the folder.
- Go to the reduction settings tab and confirm that the Q range for merge boxes in the bottom right have the values 0.13 and 0.15 and the tickbox is checked.
- Click 1D reduce plot first_time_hab, first_time_main and first_time_merged on a single graph. 
- Confirm that the merged data is identical to the main curve until 0.13, identical to the hab curve above 0.15 and an intermediate value inbetween.
- Open up the new GUI and confirm you get the same graphs if the reduction is run there. 




Partially Fixes #21776 . 

**Release Notes** 
Added to release notes


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
